File: WebSite with CosmosDB/server/Basketball/Microservice/Infrastructure/BasketballService.cs
namespace NBA.Infrastructure
{
    internal class BasketballService
    {
        protected static BasketballResponse<T> Success<T>(T value, string message = null)
        {
            return new()
            {
                Message = message,
                Code = 0,
                PayLoad = value
            };
        }

        protected static BasketballResponse<T> NotFound<T>(string message)
        {
            return new()
            {
                Message = message,
                Code = 128
            };
        }

        protected static BasketballResponse<T> Error<T>(string message)
        {
            return new()
            {
                Message = message,
                Code = 1
            };
        }
    }
}

File: Vue with MongoDB/server/Basketball/Microservice/src/main/java/com/nba/microservice/infrastructure/BasketballService.java
package com.nba.microservice.infrastructure;

public class BasketballService {

    protected static <T> BasketballResponse<T> success(T value, String message) {
        BasketballResponse<T> response = new BasketballResponse<>();
        response.setMessage(message);
        response.setCode(0);
        response.setPayLoad(value);
        return response;
    }

    protected static <T> BasketballResponse<T> notFound(String message) {
        BasketballResponse<T> response = new BasketballResponse<>();
        response.setMessage(message);
        response.setCode(128);
        return response;
    }

    protected static <T> BasketballResponse<T> error(String message) {
        BasketballResponse<T> response = new BasketballResponse<>();
        response.setMessage(message);
        response.setCode(1);
        return response;
    }
}